middle_end/flambda2/to_cmm/to_cmm.ml

```diff
@@ -49,7 +50,8 @@ let flush_cmm_helpers_state () =
   match Cmmgen_state.get_and_clear_data_items () with
   | [] ->
     let cst_map = Cmmgen_state.get_and_clear_constants () in
-    Misc.Stdlib.String.Map.fold aux cst_map []
+    let _res, items = Misc.Stdlib.String.Map.fold aux cst_map (res, []) in
```

Comment:
I'd prefer to return the `_res` here and continue explicitly ignoring it later. The current code makes it too easy to forget the `res`